### DISCUSSION TITLE
Disable Receipt Mails / Polls

### DISCUSSION THREAD

**Discussion by NoUseForUsernames** [alfio-event/alf.io#1404] [General]
Hi,
is there a way to disable the receipt mails? We're only using this for private events, so we don't need those mails that get sent out at the same time as the tickets. 

Also, i haven't quite figured out how to use polls. Activated them in the event but found no way to access them as a guest

Kind regards and thanks for your work!

**Comment by cbellone** [comment]
Hi @NoUseForUsernames ,

at the moment there is no way to selectively disable emails. 

However, the reservation confirmation email will NOT be sent if all the following conditions are verified:

- if your event is free of charge, and
- if there is only one ticket in the reservation, and
- the contact email address and the ticket holder email address are the same, and
- The following option is NOT disabled
![image](https://github.com/user-attachments/assets/b418cbb3-2ed8-4ff0-91ea-5feb471a1b90)

> Also, i haven't quite figured out how to use polls. Activated them in the event but found no way to access them as a guest

At the moment, attendees can access polls by using their name badge. [Alf.io-PI](https://github.com/alfio-event/alf.io-PI) will print their access code on the label.
This was a specific (sponsored) feature developed for a customer. We might re-evaluate this feature in the future, as we are not entirely sure it's been used since.

Celestino

P.S.: thank you for your support on OpenCollective!

**Comment by NoUseForUsernames** [comment]
Thanks for your answer. 

Concerning mails:
Are you planning to implement an option to disable ticket assignment? If i recall correctly, you can jsut disable force assignment at reservation time. Is there a way to disable this completely and send out all tickets to one mail in bulk? We just need the names of the buyers for our project.

Best
No
